src/util/accountUtils.ts

```diff
@@ -201,7 +202,7 @@ const getAccountDetail = (senderAddress: string, walletPassword: string): Accoun
 }
 
 const linkAddressToNamespaceTransaction = (namespacesID: string, linkType: string, namespacesAddress: string) => {
-  const transactionBuilder = new BuildTransactions(networkType, Hash);
+  const transactionBuilder = AppState.buildTxn
   const linktype = (linkType == 'Link') ? AliasActionType.Link : AliasActionType.Unlink;
   const namespacesid = new NamespaceId(namespacesID);
   const linkNamesapceAdd = Address.createFromRawAddress(namespacesAddress);
```

Comment:
variable wrong spelling, btw, can stick to this format ? 
linktype -> linkType
namespaceid -> namespaceId/ namespaceID
namespace not namespaces, the plural and singular is important

if encounter situation like createABCLink -> createABC_Link

if you have duplicate, use thisLinkType/ myLinkType / tempLinkType